main/guides/getting-started/explainer-how-to-make-an-offer.md

```diff
@@ -1,87 +1,11 @@
 # Making an Offer
-As you saw in the `dapp-offer-up` tutorial you could use the dapp UI to make an offer on up to three items. Let's take a look at how making offers works in this dapp.
+As you saw in the `dapp-offer-up` tutorial you could use the dapp UI to make an offer on up to three items. 
```

Comment:
maybe link to dapp-offer-up tutorial as a simple convenience here, just in case they end up here first?

Reply:
It is a subsequent page in a tutorial step. Tutorial shouldn't be there as it is right there on the left side of it.